src/tags-input.js

```diff
@@ -116,6 +116,25 @@ tagsInput.directive('tagsInput', function($timeout, $document, tagsInputConfig)
         return SUPPORTED_INPUT_TYPES.indexOf(type) !== -1;
     }
 
+    /**
+     * validate comma key
+     *
+     * Event properties to verify ','
+     * Chrome: keyIdentifier: "U+002C"
+     * FF: key
+     * IE: char|key
+     * Safari: -
+     *
```

Comment:
These are the only properties that differ between pressing `<` and `,` so i think there's no other way to fix this.